mm2src/lp_swap/maker_swap.rs

```diff
@@ -1119,6 +1117,21 @@ impl MakerSwapEvent {
             | MakerSwapEvent::TakerFeeValidateFailed(_)
             | MakerSwapEvent::TakerPaymentValidateFailed(_))
     }
+
+    fn is_error(&self) -> bool {
```

Comment:
How about
```rust
fn is_success(&self) -> bool { matches!(...) }

fn is_error(&self) -> bool { !self.is_success() }
```

I mean to replace the `matches` statement into the `is_success` method without the `!` operator. I think this `is_success` method will be useful.

Reply:
Yeah, it seems useful indeed. I'm going to add it now.

Reply:
@sergeyboyko0791 Done, please review again :slightly_smiling_face: